itl/plugins-contrib.d/storage.conf

```
@@ -102,3 +102,35 @@ object CheckCommand "ceph" {
 		}
 	}
 }
+
+object CheckCommand "btrfs" {
+    import "plugin-check-command"
+    [ "sudo", PluginDir + "/check_btrfs" ]
```

`command = ` is missing, thereby the CI tests are failing.

```suggestion
    command = [ "sudo", PluginDir + "/check_btrfs" ]
```

---

Sorry for the pebkac, fixed by ef0c68f